eox_core/edxapp_wrapper/backends/users_j_v1.py

```diff
@@ -155,6 +157,9 @@ def create_edxapp_user(*args, **kwargs):
     except Exception:  # pylint: disable=broad-except
         errors.append("No comments_service_user was created")
 
+    # Announce registration through API
```

Comment:
@mariajgrimaldi  The signal will be sent If the [transaction](https://github.com/eduNEXT/eox-core/blob/master/eox_core/edxapp_wrapper/backends/users_j_v1.py#L133) fails. Did you test what happens in this case?